app/uk/gov/hmrc/timetopayproxy/models/GenerateQuoteRequest.scala

```diff
@@ -36,17 +36,36 @@ import java.time.LocalDate
 
 import play.api.libs.json.Json
 
-final case class AdHoc(adHocDate: LocalDate, adHocAmount: BigDecimal)
+final case class ChannelIdentifier(channelIdentifier: String) extends AnyVal
 
-object AdHoc {
-  implicit val format = Json.format[AdHoc]
+object ChannelIdentifier extends ValueTypeFormatter {
+  implicit val format =
+    valueTypeFormatter(ChannelIdentifier.apply, ChannelIdentifier.unapply)
 }
 
+final case class Plan(
+                      quoteType: QuoteType,
+                      quoteDate: LocalDate,
```

Comment:
Is this a mandatory field ? Worth having a clarification with business as I remember numerous discussions around it . Thanks